docker/vep/Dockerfile.GRCh37

```diff
@@ -0,0 +1,36 @@
+FROM konradjk/vep85_loftee:1.0.3
```

Comment:
I do not want a dependency on Konrad's DockerHub image.

Let's just copy [his Dockerfile](https://github.com/konradjk/loftee/blob/hail_grch37/hail_files/Dockerfile_vep) into this repo and replace the base with hail-ubuntu. Then we can use `hail-apt-get-install` to simplify some of the stuff below. Relatedly, I reject building Python from source. The hail-ubuntu image should have a reasonable version of Python.

Reply:
The issue is that Konrad's image is built from an old ensembl image containing vep that has an old version of Python because it's Ubuntu 16.04. I don't want to mess with putting in a different version of VEP as the gnomad team is going to reject that. I don't want to be in the business of me personally having to validate any new version of VEP that we upgrade to and have to support. Also, who knows if Konrad's loftee code will still work.

I think that if we want to do this properly, then we should pull in someone else to either commit to helping out and testing the new image or build it themselves.

Reply:
@konradjk Can you chime in on this conversation?